Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/ThresholdedRandomCutForest.java

```diff
@@ -127,8 +128,7 @@ public ThresholdedRandomCutForest(Builder<?> builder) {
         // to be used later
         preprocessorBuilder.randomSeed(builder.randomSeed.orElse(0L) + 1);
         preprocessorBuilder.dimensions(builder.dimensions);
```

Comment:
Question on line 127: 

preprocessorBuilder.timeDecay(builder.transformDecay.orElse(1.0 / builder.sampleSize));

Does that mean timeDecay cannot be set by user as we only read transformDecay?

Reply:
Let me change the name as well -- Preprocessor was using names from RCF2.0 era. Given that we are moving that, this may be a good time to change the name (state classes remain unchanged for backward compatibility).

Reply:
Done